drivers/wifi/nrf700x/Kconfig.nrf700x

```diff
@@ -301,4 +301,14 @@ config NRF700X_RSSI_STALE_TIMEOUT_MS
 	  value as driver does not store it and requires RPU to provide the
 	  info.
 
+config BSS_IDLE_MGMT_OFFLOAD
+	int
```

Comment:
This can just be a bool and IMHO this need not be exposed to users, we can just decide this in the code.

Reply:
In that case, will remove this from here and define it as a macro in header.

Reply:
Removing this since UMAC is yet to add support for this.